node-notes-cli/app.js

```diff
@@ -0,0 +1,42 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+const notesStringify = JSON.stringify(data.notes, null, 2);
+const objectNotes = JSON.parse(notesStringify);
+
+const dataStringify = JSON.stringify(data, null, 2);
+const dataObject = JSON.parse(dataStringify);
+
+if (process.argv[2] === 'read') {
+  for (const property in objectNotes) {
+    console.log(`${property}: ${objectNotes[property]}`);
+  }
+} else if (process.argv[2] === 'create') {
+  const currentId = dataObject.nextId - 1;
+  dataObject.notes[currentId + 1] = process.argv[3];
```

Comment:
You can leave `currentId` as `data.nextId` without the subtraction and then skip `+ 1` on line 16.